.eslintrc.json

```diff
@@ -147,6 +147,21 @@
     "strict": "error",
     "unicode-bom": ["error", "never"],
     "valid-jsdoc": "error",
-    "wrap-iife": ["error", "inside"]
+    "wrap-iife": ["error", "inside"],
+    "space-unary-ops": ["error", {
+      "words": true,
+      "nonwords": false
+    }],
+    "space-infix-ops": ["error", {
+      "int32Hint": false
+    }],
+    "comma-spacing": ["error", {
+      "before": false,
+      "after": true
+    }],
+    "keyword-spacing": ["error",{
+      "before": true,
+      "after": true
+    }]
```

Comment:
Do we need to pass in the optional arguments? The arguments that you passed are defaults so unless we are modifying them, I don't think we need to pass them.

Reply:
agreed, the linter has the same results (after pruning the last fixup commit) with this syntax:
```
    "space-unary-ops": "error",
    "space-infix-ops": "error",
    "comma-spacing": "error",
    "keyword-spacing": "error"
```